blockchain/init_derive_sha.go

```diff
@@ -21,6 +21,13 @@ import (
 	"github.com/klaytn/klaytn/params"
 )
 
+// DeriveSha will depend on ChainConfig.DeriveShaImpl.
```

Comment:
```suggestion
// InitDeriveSha will depend on ChainConfig.DeriveShaImpl.
```

Reply:
I intended to say
"InitDeriveSha will make DeriveSha and EmptyRootHash depend only on ChainConfig.DeriveShaImpl."
I will fix the comments.

Reply:
Fixed 2931efff.